### DISCUSSION TITLE
Feature/Task/Bug tracking

### DISCUSSION THREAD

**Discussion by JeremyTHolland** [DavidRieman/WheelMUD#22] [General]
Do we want to start a project in the Projects tab to enter features, bugs, etc in a cohesive way? I'm in favor of some kind of agile project tool to keep details orderly and tasks visible. Discussions are great to get initial feedback on some kind of initiative but then turning that discussion into a user story would be the next step in my opinion. Thoughts?

Update: Or would issues be the right place to manage tasks? With a name like "Issues" it sounds to me like its used for bug tracking. So now I'm confused about the distinction between the Issues and Projects section. More information is needed on my end for clarity. To the Google!

**Comment by DavidRieman** [comment]
I actually hadn't turned on or used the "Projects" feature on any GitHub repository yet, but I've just now turned it on here to have a look.

It does appear that GitHub has been closing the gap between what's available on hand, and the tools I've wanted and we've sometimes had past years for WheelMUD (but strewn about across multiple tools/sites).  Which is great.

In the past we had used Issues to also track feature / work item level tracking and not just bugs, _but_ it does look like we can try reserving Issues for true Issues/Bugs, and try out this Projects tab for breaking down road map / features / stories / whatever the terms are that it encourages.  (Issues always felt a bit chaotic in that regard, jumbling everything together there.)  I'll research a bit more about the Projects tab and see if I can convert at least the high level roadmap into some initial items that don't fight the GitHub model here.  :)
